plugins/npm/nexus-npm-repository-plugin/src/main/java/com/bolyuba/nexus/plugin/npm/service/internal/orient/OrientMetadataStoreLifecycle.java

```diff
@@ -53,7 +53,14 @@ public void on(final NexusInitializedEvent e) throws Exception {
 
   @Subscribe
   public void on(final NexusStoppedEvent e) throws Exception {
-    orientMetadataStore.stop();
-    Orient.instance().shutdown();
+    log.debug("Asking npm metadata store to stop...");
+    try{
+      orientMetadataStore.stop();
+    } catch( Exception ex ){
+      log.warn("Problem stopping the npm metadata store", ex);
+    } finally {
+       log.debug("Asking Orient to shutdown...");
```

Comment:
I added the logging because shutdown does wait for 2 minutes before logging.

```
engineLock.writeLock().lock();
    try {
      if (!active)
        return this;

      active = false;

      workers.shutdown();
      try {
        workers.awaitTermination(2, TimeUnit.MINUTES);
      } catch (InterruptedException e) {
      }

      OLogManager.instance().debug(this, "Orient Engine is shutting down...");
```